### DISCUSSION TITLE
Use bearer token instead of username / password for authentication

### DISCUSSION THREAD

**Discussion by Nakurai** [earthaccess-dev/earthaccess#479] [Ideas]
Hello!

The earthdata portal offers the possibility to generate bearer tokens to access the data. Would it be possible to add a new authentication strategy called `token`? It could read the token from an environment variable. It would help  with deploying more secure applications.

**Comment by betolink** [comment]
Yes!! this should be supported soon. It's safer and preferable than using our credentials.

**Reply by mfisher87** [reply]
New: https://github.com/nsidc/earthaccess/issues/484

Related: https://github.com/nsidc/earthaccess/issues/415
